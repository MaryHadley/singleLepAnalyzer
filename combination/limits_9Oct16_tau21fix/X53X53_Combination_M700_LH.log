000 (1295366.000.000) 10/09 22:41:04 Job submitted from host: <192.168.63.249:41533>
...
001 (1295366.000.000) 10/09 22:41:26 Job executing on host: <192.168.63.226:40730>
...
006 (1295366.000.000) 10/09 22:41:34 Image size of job updated: 187452
	86  -  MemoryUsage of job (MB)
	87760  -  ResidentSetSize of job (KB)
...
006 (1295366.000.000) 10/09 22:46:34 Image size of job updated: 283460
	145  -  MemoryUsage of job (MB)
	147676  -  ResidentSetSize of job (KB)
...
006 (1295366.000.000) 10/10 00:01:41 Image size of job updated: 283460
	145  -  MemoryUsage of job (MB)
	147688  -  ResidentSetSize of job (KB)
...
006 (1295366.000.000) 10/10 17:08:25 Image size of job updated: 297692
	157  -  MemoryUsage of job (MB)
	160480  -  ResidentSetSize of job (KB)
...
005 (1295366.000.000) 10/10 17:13:13 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 18:30:24, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 18:30:24, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1592  -  Run Bytes Sent By Job
	306  -  Run Bytes Received By Job
	1592  -  Total Bytes Sent By Job
	306  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31866        1   3824811
	   Memory (MB)          :      157     3072      3072
...
