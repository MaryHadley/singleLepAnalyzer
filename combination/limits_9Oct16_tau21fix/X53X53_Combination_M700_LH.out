/var/lib/condor/execute/dir_18502
Total number of likelihood evaluations: 1100000099
Total number of likelihood evaluations: 11000099
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M700_LH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M700_LH/quant-toys:0-sig-8058eb4011.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M700_LH/quant-data-sig-6371129b53.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M700_LH
