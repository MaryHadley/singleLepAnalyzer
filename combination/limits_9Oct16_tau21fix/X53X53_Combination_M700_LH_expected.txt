# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0    0.13456   0.067562     0.2842  0.092741    0.19817
